src/commands/plugins/source.ts

```diff
@@ -29,9 +29,8 @@ export default class PluginsSource extends Command {
   plugins = new Plugins(this.config);
 
   async run() {
-    const remoteCommands: string[] = FeatureFlag.getSingleton()
-      .getFeatureFlagInfo<string[]>("REMOTE_COMMANDS")
-      .sort();
+    const remoteCommands: Array<{ name: string, description: string }> = FeatureFlag.getSingleton()
```

Comment:
```suggestion
    const remoteCommands = FeatureFlag.getSingleton()
```

Avoid explicating types that can be inferred.